cluster/gce/configure-vm.sh

```diff
@@ -668,7 +668,7 @@ function create-salt-master-auth() {
     mkdir -p /srv/salt-overlay/salt/kube-apiserver
     (umask 077;
       echo "${KUBE_BEARER_TOKEN},admin,admin" > "${KNOWN_TOKENS_FILE}";
-      echo "${KUBELET_TOKEN},kubelet,kubelet" >> "${KNOWN_TOKENS_FILE}";
+      echo "${KUBELET_TOKEN},system:node:node-name,uid:kubelet,system:nodes" >> "${KNOWN_TOKENS_FILE}"
       echo "${KUBE_PROXY_TOKEN},kube_proxy,kube_proxy" >> "${KNOWN_TOKENS_FILE}")
```

Comment:
this needs to be done in a coordinated way, that takes into account the following:
* updating existing clusters where this file already exists
* switching the other users in this file as well (see https://github.com/kubernetes/kubernetes/pull/39537)